.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.17
```

Comment:
hmm I assume this is for asdf? Can you remove this file?

Reply:
yes, asdf-related, removed